elfi/visualization.py

```diff
@@ -1,3 +1,7 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from collections import OrderedDict
+
 from elfi.model.elfi_model import ElfiModel, NodeReference
 
 
```

Comment:
Yes, definitely. But maybe better to leave that until we have a better picture of what is needed.